.envrc

```diff
@@ -1 +1,5 @@
 use flake
+
+watch_file pkgRio.nix
+
+eval "$shellHook"
```

Comment:
May I ask what this does does?

Reply:
`shellHook` is documented here: https://nixos.org/manual/nix/stable/command-ref/nix-shell#description

Basically, some derivations define that as a variable which the dev shell should evaluate to perform additional setup. For example, when using `pre-commit-hooks.nix` they use `shellHook` to configure the `pre-commit-hooks` when entering the shell.

Upon checking, it seems explicitly saying `eval "$shellHook"` is no longer needed with `use flake` as direnv now does it correctly. I'll remove it :)

Reply:
> shellHook is documented here: https://nixos.org/manual/nix/stable/command-ref/nix-shell#description

Oh, I thought it was a feature from `direnv` itself so I was wondering why it was nowhere documented in `direnv`. Thank you for the explanation.